src/content/changes/highlights/2022-04-22.md

```diff
@@ -0,0 +1,23 @@
+---
+date: 2022-04-22T10:00:00
+title: Highlights for the week ending April 22, 2022
+changes_categories:
+  - Highlights
+owner:
+  - https://github.com/orgs/giantswarm/teams/sig-product
+---
+
+## General
+
+
+## Apps
+
```

Comment:
@Gacko @weatherhog 
```suggestion
- nginx-ingress-controller-app [v2.11.0](https://github.com/giantswarm/nginx-ingress-controller-app/blob/master/CHANGELOG.md#2110---2022-04-22) upgrades the ingress-nginx controller container image to [v1.2.0](https://github.com/kubernetes/ingress-nginx/blob/main/Changelog.md#120). Among other changes, this release introduces deep inspection on Ingress objects. This may increase CPU usage.
- external-dns-app [v2.10.0](https://github.com/giantswarm/external-dns-app/blob/master/CHANGELOG.md#2100---2022-04-20) updates the container image of external-dns to [v0.11.0](https://github.com/kubernetes-sigs/external-dns/releases/tag/v0.11.0)
```